src/components/editor/editor-pane/tool-bar/table-picker/table-picker.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+
+
+.table-picker-container {
+  z-index: 1111;
+
+  @import "../../../../../style/variables.light";
+
+  .table-cell {
+    border-top: 1px solid $dark;
+    border-left: 1px solid $dark;
+  }
+
+  .table-container {
+    border-bottom: 1px solid $dark;
+    border-right: 1px solid $dark;
+    display: grid;
+    grid-template-columns: repeat(10, 15px [col-start]);
+    grid-template-rows: repeat(8, 15px [row-start]);
+  }
+
+  body.dark {
+    .table-cell {
+      @import "../../../../../style/variables.dark";
+      border-top: 1px solid $dark;
+      border-left: 1px solid $dark;
+    }
+
+    .table-container {
+      @import "../../../../../style/variables.dark";
+      border-bottom: 1px solid $dark;
+      border-right: 1px solid $dark;
+      display: grid;
+      grid-template-columns: repeat(10, 15px [col-start]);
+      grid-template-rows: repeat(8, 15px [row-start]);
+    }
```

Comment:
You can use one import instead of two if you move the import one level higher.

Reply:
done